yb-voyager/cmd/endMigrationCommand.go

```diff
@@ -400,6 +408,11 @@ func deletePGPublication(msr *metadb.MigrationStatusRecord, source *srcdb.Source
 }
 
 func cleanupTargetDB(msr *metadb.MigrationStatusRecord) {
+	if !msr.ImportDataIntoTargetStarted {
```

Comment:
see line 417. Looks like we alreay have a check for TargetDBConf. 
I think everwhere, you can use `msr.TargetDBConf != nil` instead of a new flag for `ImportDataIntoTargetStarted` ?